src/process.ts

```diff
@@ -20,8 +19,23 @@ export interface IExecutionHandle {
     process: proc.ChildProcess;
 }
 
+//convert bytes to string
+function bytesToString(bytes: Uint8Array): string {
+
+    const os = require('os');
+    const iconv = require("iconv-lite");
+
+    const isWin = os.platform() === 'win32';
+    var charset = 'utf8';
+    if (isWin) {
+        charset = 'gbk';
+    }
+
+    return iconv.decode(bytes, charset).toString("utf8");
+}
```

Comment:
it's common api, why move it?

Reply:
It was used only in `process.ts`, so I moved it there.

Reply:
It is a generic interface, with the possibility of adding other interfaces in the future.

I believe that each pr should make the smallest possible changes that are only relevant to the current functionality.